core/ir/x86/decode_table.c

```diff
@@ -4604,10 +4604,9 @@ const instr_info_t prefix_extensions[][12] = {
     {INVALID, 0xf20f7337, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   }, /* prefix extension 103 */
   {
-    {REX_B_EXT,  0x900000, "(rex.b ext 0)", xx, xx, xx, xx, xx, no, x, 0},
+    {REX_B_EXT,  0x900000, "(rex.b ext 0)", xx, xx, xx, xx, xx, no, x, END_LIST},
     {OP_pause,0xf3900000, "pause", xx, xx, xx, xx, xx, no, x, END_LIST},
```

Comment:
We should figure out who is correct here:
```
$ disasm f3 41 90
Disassembling 0xf3 0x41 0x90
llvm-mc:   0xf3 0x41 0x90  xchgl %r8d, %eax
capstone:  41f3 xchg eax, r8d
bfd:       f3 41 90 rex.B pause
DynamoRIO: f3 41 90 pause
```

Reply:
xed says pause

Reply:
gdb also says pause though I thought it used the bfd code (maybe newer than my objdump):
```
=> 0x40100f <_start+15>:	rex.B pause 
```
A quick asm test shows no xchg happening on my processor at least:
```
(gdb) info reg
rax            0xdeadbeef          3735928559
r8             0x12345678          305419896
(gdb) x/3bx $pc
0x40100f <_start+15>:	0xf3	0x41	0x90
(gdb) x/1i $pc
=> 0x40100f <_start+15>:	rex.B pause 
(gdb) stepi
0x0000000000401012 in _start ()
(gdb) info reg
rax            0xdeadbeef          3735928559
r8             0x12345678          305419896
```